docs/src/pages/guides/FHIRSearchConfiguration.md

```diff
@@ -103,52 +100,81 @@ A few things to note are:
 
-The search parameter filtering feature is supported through a set of inclusion rules specified via the `fhirServer/searchParameterFilter` configuration property. The search parameter inclusion rules allow you to define a set of search parameters per resource type that should be included in the FHIR server's view of search parameters when storing resources and performing search operations. The rules also allow you to specify a wildcard for resource types and also for search parameter. The following example shows the general form for specifying inclusion rules:
+The search parameter filtering feature is supported through a set of inclusion rules specified via the `fhirServer/resources` property group in `fhir-server-config.json`. The search parameter inclusion rules allow you to define a set of search parameters per resource type that should be included in the FHIR server's view of search parameters when storing resources and performing search operations. The following snippet shows the general form for specifying inclusion rules:
 
 ```
-{
-    "fhirServer": {
-        "searchParameterFilter": {
-            "&lt;resource type1&gt;": [
-                "sp-name1",
-                "sp-name2",
-                ...,
-                "sp-namen"
-            ],
-            "&lt;resource type2&gt;": [
-                "sp-name1",
-                "sp-name2",
-                ...,
-                "sp-namen"
-            ],
-            "&lt;resource type3&gt;": [ "*" ],
-            "*": [ "*" ]
+"resources": {
+    "open": false,
+    "CarePlan": {
+        "searchParameters": {}
+    },
+    "ExplanationOfBenefit": {
+        "searchParameters": {
+            "_id": "http://hl7.org/fhir/SearchParameter/Resource-id",
+            "_lastUpdated": "http://hl7.org/fhir/SearchParameter/Resource-lastUpdated",
+            "patient": "http://hl7.org/fhir/us/carin-bb/SearchParameter/explanationofbenefit-patient",
+            "type": "http://hl7.org/fhir/us/carin-bb/SearchParameter/explanationofbenefit-type",
+            "identifier": "http://hl7.org/fhir/us/carin-bb/SearchParameter/explanationofbenefit-identifier",
+            "service-date": "http://hl7.org/fhir/us/carin-bb/SearchParameter/explanationofbenefit-service-date"
         }
+    },
+    "Patient": {
```

Comment:
Suggest having on of these resource types have "*": "*" for example.

Reply:
Since leaving out `searchParameters` entirely is equivalent to having a single entry with `"*": "*"`, I don't think it should be common to need `"*": "*"`.
I do cover the scenario where `"*": "*"` is useful below (line 147).